pkg/kubelet/server/server.go

```diff
@@ -417,6 +419,77 @@ func (s *Server) InstallDebuggingHandlers(criHandler http.Handler) {
 	}
 }
 
+// InstallDebuggingDisabledHandlers registers the HTTP request patterns that provider better error message
+func (s *Server) InstallDebuggingDisabledHandlers() {
+	glog.Infof("Adding debug disabled handlers to kubelet server.")
+
+	ws := new(restful.WebService)
```

Comment:
I am not familiar with this part. So this is just a weak suggestion:
I think we can  simply route all subrequets of `/run`, `/exec`... to debugingDisabled